src/PHPUnit/Controller/AbstractHttpControllerTestCase.php

```diff
@@ -422,30 +422,28 @@ public function registerXpathNamespaces(array $xpathNamespaces)
      *
      * @param  string $path
      * @param  bool $useXpath
-     * @return Document\NodeList
+     * @return Crawler
```

Comment:
@Ocramius 
This return type was changed.

Reply:
Sure, but it's private

Reply:
I am blind, because I was sure I had read `public`.